homeassistant/components/avri/sensor.py

```diff
@@ -90,13 +65,30 @@ def available(self):
         """Return True if entity is available."""
         return self._state_available
 
+    @property
+    def device_class(self):
+        """Return the device class of the sensor."""
+        return DEVICE_CLASS_TIMESTAMP
+
     @property
     def icon(self):
         """Icon to use in the frontend."""
         return ICON
 
-    def update(self):
-        """Update device state."""
+    @property
+    def device_info(self):
+        """Return device information."""
+        return {
+            "identifiers": {(DOMAIN, self._integration_id)},
+            "name": "Avri",
+            "manufacturer": "Avri",
+        }
```

Comment:
```suggestion
```

Since we aren't providing any properties that would link entities or be discoverable by other integrations (https://developers.home-assistant.io/docs/device_registry_index/), we shouldn't register a device.

Reply:
I initially had it without device, but I liked the sensors (there's a sensor per waste type) to be grouped. Sort of like a `group by` 😅. I misunderstood `devices` in home assistant so I used a device for that. 
I removed it.